### DISCUSSION TITLE
blog/web/pourquoi-next.js-plut%C3%B4t-que-create-react-app

### DISCUSSION THREAD

**Discussion by giscus** [TawalMc/tawaldevuniverse#2] [Announcements]
# blog/web/pourquoi-next.js-plut%C3%B4t-que-create-react-app

Pourquoi se fatiguer à faire un lot de configurations si on peut juste commencer par travailler sans se gêner?

https://tawaldevuniverse.vercel.app/blog/web/pourquoi-next.js-plut%C3%B4t-que-create-react-app

**Comment by tawaliou** [comment]
Hi you can now post comments. Thank you.
